pkg/hostpath/hpv.go

```diff
@@ -76,6 +76,18 @@ func (hpv *hostPathVolume) GetTargetPath() string {
 	defer hpv.Lock.Unlock()
 	return hpv.TargetPath
 }
+
+// TODO: document!
```

Comment:
The need to document aside, this method feels correct in that we don't want/need that extra layer of indirection in our file system mount stack when we are read only

Reply:
That snippet was a left over, removed.